modules/javafx.graphics/src/test/java/test/javafx/scene/paint/ColorTest.java

```diff
@@ -653,114 +653,114 @@ public void testWebHslaRange() {
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgb(100, 100, 100"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebRgba3Param() {
-        Color.web("rgba(100, 100, 100)");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgba(100, 100, 100)"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebRgba2Param() {
-        Color.web("rgba(100, 100)");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgba(100, 100)"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebRgba1Param() {
-        Color.web("rgba(100)");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgba(100)"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebRgba0Param() {
-        Color.web("rgba()");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgba()"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebRgbaNoParen() {
-        Color.web("rgba 100, 100, 100");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgba 100, 100, 100"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebRgbaNoCloseParen() {
-        Color.web("rgba(100, 100, 100, 0.5");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("rgba(100, 100, 100, 0.5"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebHsl2Param() {
-        Color.web("hsl(240, 50%)");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("hsl(240, 50%)"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebHsl1Param() {
-        Color.web("hsl(240)");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("hsl(240)"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebHsl0Param() {
-        Color.web("hsl()");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("hsl()"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebHslNoParen() {
-        Color.web("hsl 240, 50%, 50%");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("hsl 240, 50%, 50%"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebHslNoCloseParen() {
-        Color.web("hsl(240, 50%, 50%");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("hsl(240, 50%, 50%"));
     }
 
-    @Test(expected=IllegalArgumentException.class)
+    @Test
     public void testWebHsla3Param() {
-        Color.web("hsla(240, 50%, 50%)");
+        assertThrows(IllegalArgumentException.class, () -> Color.web("hsla(240, 50%, 50%)"));
```

Comment:
even though this change is unrelated to the feature, I agree with the decision to make it in this PR.  and its trivial, so unlikely to give us trouble merging.